src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenuItems-SignOut.cshtml

```diff
@@ -3,7 +3,7 @@
 @if (!Site.As<LoginSettings>().DisableLocalLogin)
 {
     <li>
-        <a class="dropdown-item" asp-area="@UserConstants.Features.Users" asp-controller="Account" asp-action="ChangePassword" asp-route-returnUrl="@FullRequestPath">
+        <a class="dropdown-item" asp-action="EditPassword" asp-route-id="@User.Identity.Name">
```

Comment:
you should specify the area to make sure the link work from all routes.

But this will change the Change password link for both the frontend and the backend link. The backend link should take you to reset the password via backend while the frontend link does not change and stay as is today.

Reply:
Is this menu item used for both the front & admin? If yes, we need to check if it's called from the admin because nothing should be change if the link clicked from the front

Reply:
I don't recall. But I think this menu item is visible on the front end UserMenu and also the backend UserMenu. (at least it should be on both)

Reply:
I will check now and react to other comments

Reply:
The same menu has been used for both, with that if the user has admin access we should redirect to edit password regardless from where he access the menu, otherwise it will redirected to change password